src/Certify.Tests/Certify.Core.Tests.Unit/CertifyManagerAccountTests.cs

```diff
@@ -21,6 +21,29 @@ public CertifyManagerAccountTests()
             _certifyManager.Init().Wait();
             var testCredentialsPath = Path.Combine(EnvironmentUtil.GetAppDataFolder(), "Tests", ".env.test_accounts");
             DotNetEnv.Env.Load(testCredentialsPath);
+            SetupAccount().Wait();
+        }
+
+        private async Task SetupAccount()
+        {
+            var currentAccounts = await _certifyManager.GetAccountRegistrations();
+            if (currentAccounts.Count == 0)
+            {
+                var contactRegEmail = "admin." + Guid.NewGuid().ToString().Substring(0, 6) + "@test.com";
+                var contactRegistration = new ContactRegistration
+                {
+                    AgreedToTermsAndConditions = true,
+                    CertificateAuthorityId = "letsencrypt.org",
```

Comment:
Let's do something in configuration so we can substitute the CA for a test one like smallstep